en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,113 @@
+    * **Package name** sholud match the package name in Packagist.
+    * **Affected versions** should be a string reprenting a range of version affected by the vulnerability.
+    * **Patched versions** should be a list version where the vulnerability was/will be patched.
+  * **Severity** should be set to *Assess severity using CVSS* and the CVSS option should be specified in the calculator.
+  * **Common weakness enumerator (CWE)** should be left blank if no suitable option can be found in the list.
+  * **CVE identifier** should be set to "Request CWE ID later".
+  * **Title** and **Description** should be descriptive enough that people can evaluate the risk without being so specific that would-be-hackers can start exploiting them right away with no efforts. 
+
+These settings can be changed later, so you can iterate over them.
+
+## Special circumstances
+
+### An issue has been reported publicly on GitHub
+
+Occasionally, some developers will open a public issue on GitHub reporting a vulnerability, either because they are unaware of the proper process or because they did not immediately grasp the security implication of the bug being reported.
+
+In those situations, the security team will assess how likely it was that the vulnerability was widely noticed based on how long the issue was opened on GitHub and how explicit the description was. If the risk of the vulnerability having been noticed is small, an owner on the Silverstripe GitHub organisation will delete the public facing issue and create a private one instead.
+
+### Handling issues on Huntr
+
+[Huntr.dev](https://huntr.dev/) is an open source vulnerability bounty platform. The security team occasionally receives vulnerability reports from Huntr. Reports from Huntr go through more or less the process as the regular reports. The security team should take the following points into consideration when working on a Huntr issue:
+
+- While you can reply via email and the response will be recorded in Huntr, who sent the response will not. To keep the discussion flowing more naturally, it is preferable to login into Huntr with your GitHub account and respond via the Huntr interface.
+- People reporting vulnerabilities via Huntr are not always very familiar with Silverstripe CMS. Don't assume that they have a lot of context on what Silverstripe CMS is used for.
+- Some Huntr reporters are just looking to make a quick buck. Review the reporter's profile to see how serious and reliable they are. Occasionally, some reporters will try to open multiple reports for the same underlying vulnerability or push for a higher CVSS score to get a bigger bounty. Do not hesitate to push back if the reporter is being unreasonable.
+- Once we "confirm" an issue in Huntr, it means that we agree the bug exist and that it's a vulnerability. Security enhancement should not be confirmed.
+- By default, Huntr will issue CVEs for issues reported through their platform. We've disabled this functionality.
```

Comment:
Does this need to be mentioned since it's already been disabled?

Reply:
Researcher will occasionally comment on the fact that there's no CVE assigned to the Huntr issue. It's a deviation from the Huntr workflow.